types/csv-stringify/index.d.ts

```diff
@@ -13,7 +14,7 @@ declare namespace stringify {
          * Order matters, read the transformer documentation for additionnal information,
          * columns are auto discovered when the user write object, see the "header" option on how to print columns names on the first line.
          */
-        columns?: string[];
+        columns?: object | string[];
```

Comment:
A `string[]` is also an object, so this typing doesn't mean anything more than just putting `object`. 
Would it be possible to make the type `columns?: { [index: string]: string } | string[]`?